plat/intel/soc/stratix10/plat_sip_svc.c

```diff
@@ -0,0 +1,381 @@
+/*
+ * Copyright (c) 2019, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <assert.h>
+#include <common/debug.h>
+#include <mmio.h>
+#include <common/runtime_svc.h>
+#include <uuid.h>
+#include <s10_mailbox.h>
+
+/* Number of SiP Calls implemented */
+#define SIP_NUM_CALLS		0x3
+
+/* Total buffer the driver can hold */
+#define FPGA_CONFIG_BUFFER_SIZE 4
+
+int current_block;
+int current_buffer;
+int current_id = 1;
+int max_blocks;
+uint32_t bytes_per_block;
+uint32_t blocks_submitted;
+uint32_t blocks_completed;
```

Comment:
Can you make them static?

Reply:
Hmm why make them static?
This should be used only locally in this c file, not outside of it.